Assets/SteamVR_Unity_Toolkit/Scripts/Controls/3D/VRTK_Chest.cs

```diff
@@ -0,0 +1,227 @@
+﻿namespace VRTK
+{
+    using UnityEngine;
+
+    public class VRTK_Chest : VRTK_Control
+    {
+        public enum Direction
+        {
+            autodetect, x, z // no support for y at this point in time
+        }
+
+        public Direction direction = Direction.autodetect;
+        private float min = 0f;
```

Comment:
one of your pet hates ;) , but could the private params be grouped at the bottom and the public params at the top as per the contrib guide? :)